collect-config-setup/fragments/configure_config_agent.sh

```diff
@@ -74,8 +74,7 @@ mkdir -p $oac_templates/var/run/heat-config
 echo "{{deployments}}" > $oac_templates/var/run/heat-config/heat-config
 
 # os-refresh-config scripts directory
-# This moves to /usr/libexec/os-refresh-config in later releases
-orc_scripts=/opt/stack/os-config-refresh
+orc_scripts=/usr/libexec/os-refresh-config
```

Comment:
Don't you break backward compatibility with changing the path? You could use symlink to assure that it will keep working with older version.